evap/staff/staff_mode.py

```diff
@@ -59,11 +58,9 @@ def update_staff_mode(request):
 
 def enter_staff_mode(request):
     update_staff_mode(request)
-    delete_navbar_cache_for_users([request.user])
```

Comment:
This works because our enter_staff_mode view will, afterwards, redirect to evaluation:index, which will do the deletion.

If, at some point, we decide to change this redirect, we will likely break it again, but the test should catch that, because re-entering after leaving should then fail, right?

Reply:
yes, i think so, too. and hopefully #1561 will make all of this obsolete :)